docs/getting-started/installation-common/scylla-web-installer.rst

```diff
@@ -27,7 +28,13 @@ You can run the command with the ``-h`` or ``--help`` flag to print information
 Installing a Non-default Version
 ---------------------------------------
 
-You can install a version other than the default.
+You can install a version other than the default. To get the list of supported
```

Comment:
Is it a list of *[supported](https://docs.scylladb.com/stable/versioning/version-support.html)* or *available* versions?

Reply:
supported